packages/apps/graph/.env.example

```diff
@@ -1,11 +1,15 @@
 CHAIN_COUNT=20
 
+# Database configuration. The default value is postgresql://devnet@localhost:5432/devnet.
+DATABASE_URL="postgresql://devnet@localhost:5432/devnet"
+
+# This variable determines the maximum calculated block confirmation depth. The default is 51.
+MAX_CALCULATED_BLOCK_CONFIRMATION_DEPTH=51
```

Comment:
Can we leave this at 11? This will make the queries faster and I feel that a number much greater than 6 is not that useful anyway

Reply:
Updated!